sklearn/metrics/scorer.py

```diff
@@ -313,12 +322,12 @@ def make_scorer(score_func, greater_is_better=True, needs_proba=False,
 
 # Standard regression scores
 r2_scorer = make_scorer(r2_score)
-mean_squared_error_scorer = make_scorer(mean_squared_error,
-                                        greater_is_better=False)
-mean_absolute_error_scorer = make_scorer(mean_absolute_error,
-                                         greater_is_better=False)
-median_absolute_error_scorer = make_scorer(median_absolute_error,
-                                           greater_is_better=False)
```

Comment:
I would rather avoid making the `sklearn.utils.deprecation.deprecated` wrapper too complex. I think it's good enough to not use it at all in this case and just use a `_deprecation_msg` attribute on `_BaseScorer` as suggested in https://github.com/scikit-learn/scikit-learn/pull/7261#discussion_r76979585.